Source/JavaScriptCore/heap/ConservativeRoots.cpp

```diff
@@ -94,11 +94,10 @@ void ConservativeRoots::genericAddSpan(void* begin, void* end, MarkHook& markHoo
     RELEASE_ASSERT(isPointerAligned(begin));
     RELEASE_ASSERT(isPointerAligned(end));
 
-    TinyBloomFilter<uintptr_t> filter = m_heap.objectSpace().blocks().filter(); // Make a local copy of filter to show the compiler it won't alias, and can be register-allocated.
     HeapVersion markingVersion = m_heap.objectSpace().markingVersion();
     HeapVersion newlyAllocatedVersion = m_heap.objectSpace().newlyAllocatedVersion();
     for (char** it = static_cast<char**>(begin); it != static_cast<char**>(end); ++it)
-        genericAddPointer(*it, markingVersion, newlyAllocatedVersion, filter, markHook);
+        genericAddPointer(*it, markingVersion, newlyAllocatedVersion, markHook);
```

Comment:
Why is this change necessary?

Reply:
No, but I thought it was a worth addition. I'll revert it.